CONTRIBUTING.md

```diff
@@ -1,174 +1,267 @@
-# How To Contribute
+# Contributions
 
-Fusion welcomes issues and pull requests from the community! However, make sure
-you've read the following first *before* opening any issues or pull requests, so
-the process can be as smooth as possible 🙂
+We love contributions and suggestions from you guys! To make sure everything goes as smoothly as possible, we've set up a few processes you can follow for different contribution types:
 
-## I have a question, or am stuck and need help using Fusion
+<!----------------------------------------------------------------------------->
+
+<details>
+<summary>I have a question, or am stuck and need help using Fusion</summary>
+<br>
 Don't open an issue to ask a question - see our
-[Get Started page](https://elttob.github.io/Fusion/tutorials/#where-to-get-help)
-to find a place to get help.
+<a href="https://elttob.github.io/Fusion/tutorials/#where-to-get-help">Get Started page</a>
+to find places to get help! Our communities are always happy to help out new people 🙂
+<hr>
+</details>
+
+<!----------------------------------------------------------------------------->
 
-## I found a security vulnerability or other sensitive issue
+<details>
+<summary>I found a security vulnerability or other sensitive issue</summary>
+<br>
 Please use our [vulnerability reporting process](./SECURITY.md). Do not openly
 disclose sensitive bugs, issues or vulnerabilities - this is dangerous!
```

Comment:
As said in https://github.com/Elttob/Fusion/issues/90#issuecomment-988330275, security.md should be a lot more precise. I think that should be part of this PR. Just the basics of "Try to contact Elttob at <email> or on Discord at Elttob#0001" is a good step forward.